src/Core/Models/Business/OrganizationUserInvite.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections.Generic;
+using Bit.Core.Models.Api;
+using Bit.Core.Models.Interfaces;
+using System.Linq;
+using Bit.Core.Models.Data;
```

Comment:
Please consider sorting your using directives.

Reply:
Thanks! All sorted 😁 , however don't need the space or "groups", just overall sorting is fine.